src/modules/rover_pos_control/RoverPositionControl.cpp

```diff
@@ -589,6 +618,20 @@ RoverPositionControl::Run()
 			v_torque_sp.xyz[1] = _act_controls.control[actuator_controls_s::INDEX_PITCH];
 			v_torque_sp.xyz[2] = _act_controls.control[actuator_controls_s::INDEX_YAW];
 			_vehicle_torque_setpoint_pub.publish(v_torque_sp);
+
+			// Normalized (When max thrust, magnitude = 1) thrust in North - East frame
+			// const Vector2f normalized_thrust_vec_2d = _act_controls.control[actuator_controls_s::INDEX_THROTTLE] * Vector2f(_local_pos.vx, _local_pos.vy).unit();
+			// normalized_thrust_vec_2d.copyTo(_local_pos_setpoint.thrust);
+
+			// Hack: For now send the throttle setpoint in Acc X value for debugging
+			_local_pos_setpoint.acceleration[0] = _act_controls.control[actuator_controls_s::INDEX_THROTTLE];
```

Comment:
TODO: Remove, after discussing correct way to implement relaying info of the throttle command.